**Source: Thesis:** Ye C. Engineering genetically controlled microbial consortia [Internet]. 2016 [cited 2019 Dec 11]. Available from: [https://scholarship.rice.edu/bitstream/handle/1911/95582/CHEN-DOCUMENT-2016.pdf?sequence=1&isAllowed=y](https://scholarship.rice.edu/bitstream/handle/1911/95582/CHEN-DOCUMENT-2016.pdf?sequence=1&isAllowed=y)
 
**.2.1. λ-red recombination**  
This method was first described by Datsenko and Wanner (_29_), and further  
optimized by Sharan et al_._ (_97_). In this study, the following modified protocol was  
followed.  
1. pSIM19/pKD46 was transformed into the working strain by following the  
protocol given in 4.1.4. The growth condition was maintained at 32℃ as either  
**96**  
plasmid will be lost at 37℃.  
2. Knock-in fragment was prepared. The length of the homolog sequence was from  
50bp to 200bp on each side which depended on the insert length. 50bp is  
enough for typical KO replacement fragment, which is less than 1kb (an  
antibiotic gene with two flp sites). If a long fragment (3-5kb) was to be knockedin, a 200bp homolog sequence would have been used. The fragment in pCH1  
(chloramphenicol resistance) plasmid was used for all gene knock-outs. DpnI  
was used to digest the PCR product. In the final cleanup step, H2O was used  
instead of elution buffer for DNA solution.  
3. The strain was prepared with pSIM19/pKD46 for electroporation (chemical  
transformation has extremely low efficiency for linear DNA). The protocol is as  
follows:  
a) The beginning steps are the same as in preparing chemical competent cells  
given in 4.1.4 (step 3), but all culturing temperatures was not higher than  
32℃. For pKD46 plasmid, 10mM arabinose was added when OD600 reached  
0.1. The growth was continued until OD600 = 0.4. For pSIM19 plasmid, the  
cells were transferred to 42℃ 220rpm for 15min until the OD600 reached  
0.35. At this stage, the cells were ready for collection.  
b) The cells were chilled on ice-water bath for 10min, centrifuged at 1000g for  
5min and the medium was discarded. The cell pellet (from 50ml medium  
culture) was resuspended in 10ml H2O. Then, centrifuged at 1000g for 5min.  
The water was discarded. This step was repeated for three times in order to  
**97**  
wash the cells. The final spun was done at 4000g for 5min. The temperature  
for all the centrifugation steps was 4℃.  
c) 200μl H2O was added for resuspension. 10μl insert DNA solution (less than  
0.5μg) was mixed with 40μl cells. The mixture was transferred to 1mm  
electroporation cuvette.  
d) The electroporator was set to 1.25kV for electroporation. Electroporation  
would fine if the pulse time is larger than 4ms. If not, cells need to be further  
diluted or less DNA fragments to be added in step c. This step was repeated  
again until the pulse time fulfills the requirement.  
e) 1ml LB medium was added immediately. And the cells were incubated at 37℃  
(pKD46)/32℃ (pSIM19) for 2h.  
f) The cells were spun down and resuspended in 200μl LB medium. Then they  
were plated as aliquots of 100μl on each LB plate which contained 5 and  
10μg/ml chloramphenicol.  
g) The plates were incubated at 37℃ for overnight.  
4. Single colonies were picked up and restreaked on LB (25μg/ml chloramphenicol)  
plate. The colony PCR protocol given under Section 4.1.5 was followed to find the  
KO/KI colony (design primers which bind to each side of target gene). Then, the  
positive colony was resuspended into 50μl LB medium. The cells were streaked  
on LB plates for single colony. The streaked plates were incubated at 37℃/42℃  
overnight. The colony was tested for growth on ampicillin (pKD46) or  
spectinomycin (pSIM19) plates. The colony should only have chloramphenicol  
**98**  
resistance.  
5. Chemically competent cells were prepared from the new KO/KI strain. pCP20  
plasmids were transformed into the new strain (see Section 4.1.4, 100μg/ml  
ampicillin at 30℃ for growth). The looping out of chloramphenicol resistance  
gene by the colony was tested by using PCR. Positive colony was incubated at  
37℃/42℃ to eliminate the pCP20 plasmid. The final strain did not have any  
antibiotic resistances.  
Notice: This KO/KI method is not scar free. The sequences of two flt sites and  
one flt sequence will remain in the genome. If work on one strain is done for  
multiple times, then the homolog sequence in the later KI fragment should be as less  
as possible to the previous scar/KI sequence. Otherwise, a small proportion of  
insertion will appear at the first insertion site