> [!caution] This page contained a drawing which was not converted.   

List of keck organisms : [Google sheets](https://docs.google.com/spreadsheets/d/1nKg2KenPX2z2vX1OHzJ9uPjpwy_4oVEg4PmJ2Ny1LxM/edit#gid=0)
 
E coli base strains present in bennett lab : [https://wiki.rice.edu/confluence/display/BIODESIGN/E.+coli+Base+Strains](https://wiki.rice.edu/confluence/display/BIODESIGN/E.+coli+Base+Strains)
    
**Useful information about a few bacteria** (IGEM 2018 : [July lab notebook](https://static.igem.org/mediawiki/2018/a/ab/T--Rice--July.pdf) : 7/1/2018)

![Machine generated alternative text:Strain Escherichia coli (MG1655, DHIOB, Nissie 1917) Corynebacte num glutamicum Shewanella oneidensls Pseudomona s putlda Sinorhizobiu m melilpgg Vibrio natriegens Bacillus subtilis Lactococcus lactis Type Gram- negative Gram- negative Gram- negative Gram- negative Gram- negative Gram- negative Gram- positive Gram- positive Phylum Proteobacteria Proteobacteria Proteobacteria Proteobacteria Proteobacteria Proteobacteria Firmicutes Firmicutes Applicatio Model gram- negative Industrial production of amino acids Bioelectro n.cs Bioremedi ation Agriculture Industrial production of various compound Model gram- positive Therapeuti cs, cheese production Kan susceptible yes yes yes yes low level resistance; look for faster growing colonies yes Neo susceptible p88 Yes prob ably prob ably Yes Yes prob ably may RSFIOIO Yes probably probably probably probably probably ](Exported%20image%2020250102022424-0.png)  

**Temperatures** (Source - IGEM 2018/[June lab notebook](https://static.igem.org/mediawiki/2018/d/dd/T--Rice--June.pdf) : 6/13/2018)  
Vmax, B. subtilis, C. glutamicum, nissle1917, DH10B, L. lactis went in 37C incubator3.  
P. putida, S. oneidensis, and S. melitoti went into 30C incubator
 
**Antibiotic : July :** (IGEM 2018 : [July lab notebook](https://static.igem.org/mediawiki/2018/a/ab/T--Rice--July.pdf) : 7/11/2018)

- More details/ polished results [[Antibiotics] - other organisms](%5bAntibiotics%5d%20-%20other%20organisms)
- MG1655, C. glutamicum, L. lactis susceptible to 50 ug/mL of kanamycina.
- Nissle1917 and S. oneidensis maybe susceptible; need to check higher concentrationsb.
- DH10B, B. subtilis, P. putida don't seem affected by 50 ug/mL kanamycinc.
- S. meliloti didn't grow at all, even in plain medium
   

**Growth rates**  
To identify growth conditions suitable for all strains and measure growth rates for parameter estimation in [modeling](https://2018.igem.org/Team:Rice/Model) , we performed OD time course measurement for all strains used in our project (Figure 1). Cells were **grown in LB at 30°C for 15 hours.** The results demonstrate that all strains are able to grow under those conditions, with _E. coli_ strains and _B. subtilis_ exhibiting the highest growth rates.

![Exported image](Exported%20image%2020250102022425-1.png)

**Figure 1: Growth rates of bacteria used in the project**
 > From <[https://2018.igem.org/Team:Rice/Results#Strains](https://2018.igem.org/Team:Rice/Results#Strains)>     

**Notes on ribosomes/16S similarity for translational compatibility**  
However, evidence exists which suggests that 16S rRNA can be transferred across organism without the loss of function4. According to Kitahara _et al._ , 16S rRNA from a variety of organisms was successfully expressed in _E. coli_ even if the similarity between the 16S sequences was as low as 80%. After comparing 16S rRNA sequences from the bacteria used in our project, we hypothesized that _E. coli_ 16S rRNA can be expressed in other gram-negative bacteria we have selected due to high sequence similarity (Figure 3). If _E. coli_ 16S rRNA fails in gram-positive bacteria, future work would involve expressing native 16S rRNA for each species.
 > From <[https://2018.igem.org/Team:Rice/Background](https://2018.igem.org/Team:Rice/Background)>  
![Exported image](Exported%20image%2020250102022427-2.png)

**Figure 3: Pairwise percent identities of 16S rRNA sequences from organisms used in the project.** The comparison of pairwise percent identities demonstrates that 16S rRNA sequences among gram-negative bacteria have > 80% similarity.
 
- Kitahara, Kei, Yoshiaki Yasutake, and Kentaro Miyazaki (2012). Mutational Robustness of 16S Ribosomal RNA, Shown by Experimental Horizontal Gene Transfer in Escherichia Coli.” Proceedings of the National Academy of Sciences of the United States of America 109 (47), 19220–25.
    
positive

Actinomycetota

Table with references for organisms : [S079 (Pl) - redo Conjugation of pPK051/pPK080](S079%20(Pl)%20-%20redo%20Conjugation%20of%20pPK051\pPK080%20-%20all%20organisms)

|   |   |   |   |   |   |   |   |
|---|---|---|---|---|---|---|---|
==#==1|2|3|4|==5== ==6==7|8|9|10|
==Organism==Ec MG1655|L.lactis|1. glutamicum|Vmax|B. subtilis 168|A.baylyi ADP1-ISx/  <br>[bMRB103](https://wiki.rice.edu/confluence/display/BIODESIGN/Other+Bacterial+Strains)||P. putida|S. oneidensis|==S. meliloti==
==Media, reference==LB|+[Ascorbic acid, Mg](https://www.sigmaaldrich.com/US/en/product/sial/63016)SO4 ([M17](https://journals.asm.org/doi/10.1128/jb.00533-10) media)|; [2x TY](https://www.frontiersin.org/articles/10.3389/fmicb.2019.01648/full#h3)|LB-V2 / LB||[LB-miller](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7192602/)||||==+[CaCl2,MGSO4,](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0206781#sec007) CoCl2==
==Temp==30, 37C|30C|30C|30, 37C||30C||30C?|30C?|==30C==
==Plate #==1|3|3|2|2, fast|2||1|1|==3 + "==