|   |   |
|---|---|
|Applications|Producing sample for amplicon RNA-sequencing : NGS|
|Procedure|[NEB LunaScript](https://www.neb.com/products/e1555-lunascript-multiplex-one-step-rt-pcr-kit#Protocols,%20Manuals%20&%20Usage) multiplex 1-step RT-PCR, planning  <br>Rxn composition<br><br>\|<br>\|<br>==Rxn vol== ==25 ul== ==Final conc==<br>\|Water\|10 ul\|<br>\|Buffer mix\|5 ul\|<br>\|Enzyme mix\|1\|<br>\|Primer pair mix\|5 ul\|0.5 uM / primer\|<br>\|Template (RNA)\|4 ul\|Dilute 10x; < 1 ug\|<br><br>- Primer pair mix (100 ul aliquot) : 5 ul / rxn => 2.5 uM/primer in mix : 2.5 ul each primer from 100 uM + 95 ul IDTE buffer.<br>- (Matt Dysart) _Using less template volume => less chances of chimera formation!_<br><br>  <br><br>Reaction conditions<br><br>\|<br>\|<br>==CYCLE STEP== ==TEMP== ==TIME== ==CYCLES==<br>==Initial melt secondary structure== ==60C== ==1 min====  <br>==<br>\|Reverse Transcription\|55°C*\|10 minutes\|1\|<br>\|RT Inactivation/ Initial Denaturation\|98°C\|1 minute\|1\|<br>\|Denaturation\|98°C\|10 seconds\|==35–40==<br>\|Annealing\|60–72°C**\|10–20 seconds (singleplex)\|==  <br>==<br>\|Extension\|72°C\|20–30 second/kb\|==  <br>==<br>\|Final Extension\|72°C\|5 minutes\|1\|<br>\|Hold\|4°C\|∞\|<br><br>  <br>> From <[https://www.neb.com/protocols/2021/09/08/lunascript-multiplex-one-step-rt-pcr-kit-protocol-neb-e1555](https://www.neb.com/protocols/2021/09/08/lunascript-multiplex-one-step-rt-pcr-kit-protocol-neb-e1555)>  <br>  <br>> Detailed [instruction manual](https://www.neb.com/-/media/nebus/files/manuals/manuale1555.pdf?rev=1b8c76694a8747be82933e0f2bb3f32f&hash=3E5AE68E87C891FA1520E2304190B1BF)  <br>  <br><br>The kit contains : Luna RT (thermostable, warmstart) ; Q5 (hot start)<br><br>- [Luna RT](https://www.neb.com/luna/optimize-rt-qpcr) is _in-silico_ designed; with reversible aptamer|
|Initial melting|NEB help says :  <br>The LunaScript RT could be used up to 60C, but will lose activity fairly quickly above that, at 65C. You could use 60C for the RT step, or incorporate a 2min 60C step ahead of the 55C step if you'd like. Please let me know if you have any other questions.<br><br>  <br><br>Question:  <br>I'm noticing some non-specific products and some without the desired product and was wondering about adding a initial 65 C step for 2 min to enhance primer specifity before RT of 55C 10 min…I think the major mispriming is happening at the RT stage.|
|Optimizations-tweaks|Initial **denaturation** of **secondary structures** : _don't exceed 60C ; can do for 2 mins_ (tech support)<br><br>  <br><br>Does **diluting** the RNA to **remove inhibitors** improve amplification?<br><br>- Poor RNA purity could cause : Low product yield, non-specific bands (_NEB luna_ [instruction manual](https://www.neb.com/-/media/nebus/files/manuals/manuale1555.pdf?rev=1b8c76694a8747be82933e0f2bb3f32f&hash=3E5AE68E87C891FA1520E2304190B1BF))<br>- (_maybe RT-PCR is different_) But we previously seen qPCR inhibition effect mostly from the residual buffer/cations from DNAse removal step [q014b (S) 10/9/21](q014b%20(S)%2010/9/21%20-%20DNAse%20removal%20inhibition)<br><br>  <br><br>On the same note, here's an elaborate response from NEB upon asking about inhibition/optimizations : [NEB on inhibition : 20/2/24](1-step%20RT-PCR%20NEB%20LunaScript%20(for%20long%20amplicons)<br><br>  <br>  <br><br>Can we remove spurious primer chimeras by **adding DMSO** ([B.sub_U64_S055 sanger seq]((F,%20L)%20-%20confirm%20Bacillus%20subtilis)) -- _Check if any papers have messed with RT reactions with additives:_<br><br>- 5% DMSO seems to be ok. "DMSO at concentrations higher than 15% inhibited the reverse transcriptase reactions"<br>> From <[https://pubmed.ncbi.nlm.nih.gov/8520520/](https://pubmed.ncbi.nlm.nih.gov/8520520/)>  <br>- This paper shows that 6% DMSO is optimal for RT-PCR with recombinant _Thermus thermophilus_ DNA pol (rTth): Sidhu, Maninder K., Mei-June Liao, and Abbas Rashidbaigi. "Dimethyl sulfoxide improves RNA amplification." Biotechniques 21.1 (1996): 44-47.<br><br>[Biotechniques, 1996](https://www.future-science.com/doi/pdf/10.2144/96211bm08)<br><br>  <br><br>> From <[https://pubmed.ncbi.nlm.nih.gov/8520520/](https://pubmed.ncbi.nlm.nih.gov/8520520/)>  <br>  <br><br>DMSO will change the biases in the 16S primer binding -- _the bias will be consistent in 16S and RAM so it's not an issue_<br><br>- Few 16S studies are adding 5% DMSO so this should be fine : [plos one](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0227434) ; [microbial methods 2018](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6375304/)<br><br>  <br>NEB tech support suggestions from Rachel<br><br>- Annealing : Static 66C ; short : 10 s<br>- To prevent primer dimer : assemble on ice, put onto pre-heated thermocycler<br>- Non spec : Also run a no-RT control -- could be products from the DNA template<br>- DMSO might help ; can also try the GC enhancer -- _Could also throw into the current mix?_|
|NEB on inhibition : 20/2/24|On 20/2/24 9:35 am, TSHelp@neb.com wrote:<br><br>\|   \|   \|<br>\|---\|---\|<br>\|**Dear Prashant,**<br><br>  <br><br>Thank you for contacting NEB Technical Support  <br>**All polymerases, including reverse transcriptases, are sensitive to certain inhibitors** present in samples. **Generally, we do not recommend more than 20% v/v of sample per final volume for this reason**. Additionally, as is the case with Q5, some polymerases have strict buffering conditions, and by using a larger volume of sample the reaction is no longer buffered for ==optimal== polymerase activity. ==I can't say what the impact of these inhibitors are across amplicons because this would be highly reaction specific and likely require real-time analysis.==<br><br>  <br><br>==The inhibition you observe could be happening at the cDNA synthesis or amplification levels, but without== testing different combinations of RTs and DNA polymerases (a large and likely unproductive task in my opinion) ==this is hard to know. I would not advise spiking in a different RT because it would only add another variable to the reaction, so it would be best to stick with a single enzyme. There are pros and cons in a one-step or two-step workflow, but two-steps give you more freedom to optimize each reaction and the cDNA can be used for multiple targets.==<br><br>  <br><br>==I would recommend trying to== **concentration your purified sample** ==as much as possible to== **minimize input volume** ==and/or a two-step method using our most inhibitor-tolerant RT and DNA polymerases:==<br><br>- ==Induro RT (====M0681====)==<br>    <br>    - Unfortunately, we do not offer samples of this product.<br>- ==Q5 Blood Direct 2X Master Mix (====M0500====)==<br>    <br>    - ==I can send you a sample of this to try if you would like- just provide your shipping address.==<br><br>==I hope you find this information useful. If you need any additional information or assistance troubleshooting I am happy to help.==  <br>If you have any further questions, please reply to this email.<br><br>  <br><br>Best wishes,  <br>Krist, NEB tech support\|<br><br>  <br><br>-----  <br>Cost analysis<br><br>\|   \|   \|   \|   \|<br>\|---\|---\|---\|---\|<br>\|NEB 1-step\|NEB inh tolerant 2-step\|[Invitrogen 1-step](https://www.thermofisher.com/order/catalog/product/12594025)\|<br>\|Cost/rxn (trial)\|**Is this worth a try?**\|<br>\|Cost /rxn (large)\|965 / 250 = 3.86\|- Induro: 400 / 50 = 8<br>    <br>    - _+ more reagents needed_<br>- Q5 blood direct: 504/500 = 1\|1,044/100 = 10.5\||
|Alternate products|NEB has a new RT enzyme for longer products (can go upto 60C) - [Induro reverse transcriptase](https://www.neb.com/products/m0681-induro-reverse-transcriptase?utm_source=monthly-newsletter&utm_medium=email&utm_campaign=--productpromotion-amplification-2022-10&utm_content=m0681#Product%20Information)<br><br>  <br>- Discussed with Esther about using [Superscript](https://www.thermofisher.com/order/catalog/product/12594025?us&en#/12594025?us&en) IV: 1-step RT-PCR protocol that might be cleaner (_they already use this for the Arctic sequencing method.._) // _This is 3x more expensive than NEB!_<br>    <br>    - Although recently, they switched to lunamix, don't know why : Presumably cost difference?<br>  <br><br>- The polymerase has a processivity domain attached, I don't think the PCRbio mixes have that..<br><br>  <br>  <br><br>How does this compare to the currently used PCRbio 1-step mixes? (probe or otherwise)|